src/shoji.py

```diff
@@ -0,0 +1,23 @@
+"""Shoji helper functions."""
+
+# These have been extracted from server/tests/controllers/base.py
+# I imagine they should go into pycrunch.
+
+def as_catalog(x, **kwargs) -> dict:
```

Comment:
@aminusfu Are we ready to drop python 2.x?

This is a new stand alone module, so nobody is really importing it.

Reply:
With respect to line 4. This is in munchkin. I will delete this line.

Reply:
@jjdelc, jumping onto your comment, pls check in with me before deciding to drop python 2.x. We're still trying to dig ourselves out of it.

Reply:
Removing the Python3ism `-> dict` in line 6 is easy to do. I will make these changes tomorrow.